### DISCUSSION TITLE
Display total in tooltip of stacked area chart

### DISCUSSION THREAD

**Discussion by adeel-usmani** [apexcharts/apexcharts.js#3493] [General]
Is there anyway I can get `total` number of data along with the data in the tooltip.
Like in the given picture if I want to display the total in the tooltip as well (addition of all these 3 data points)
![image](https://user-images.githubusercontent.com/102028432/202411042-2be56180-802a-4a7c-8b27-af3023b46f5f.png)

**Comment by sebj54** [comment]
I've tried many things and I didn't find a solution to the same problem. 

I think the only way is to use a custom tooltip render function but I can't find a way to reuse the default tooltip render function either.

If anyone has any hint on this, it would be very helpful!

**Comment by toaster-robotics** [comment]
This isn't exactly what you were asking for, but it was good enough for me.

You can customize the "x" text on the tooltip. It'll change it from **17 Feb** to **17 Feb (Total: 91)**

```html
tooltip: {
            shared: true,
            intersect: false,
            x: {
              formatter: function (
                value,
                { series, seriesIndex, dataPointIndex, w }
              ) {
                const series1Y = series[0][dataPointIndex] || 0;
                const series2Y = series[1][dataPointIndex] || 0;
                const stackedY = series1Y + series2Y;

                return `${
                  data.dates[dataPointIndex]
                } (Total: ${stackedY.toFixed(2)})`;
              },
            },
          },
```
